KNOWN_ISSUES.md

```diff
@@ -45,12 +45,29 @@
   OKD-specific catalog with community versions of mentioned operators is in progress.
 
 ## [Systemd-resolved is not configured properly during install](https://github.com/openshift/okd/issues/690)
-  **Effected Versions:**  4.7.0-0.okd-2021-06-13-090745 - 4.7.0-0.okd-2021-06-13-090745
+  **Affected Versions:**  4.7.0-0.okd-2021-06-13-090745 - 4.7.0-0.okd-2021-06-13-090745
 
   **Description:** With the newest stable release, systemd-resolved is misconfigured due to a missing directory.  This prevents nodes from installing properly.
   
-  **Workaround** from each affected node do the following
-* sudo su -
-* mkdir /etc/systemd/resolved.conf.d
-* reboot
+  **Workaround:** From each affected node do the following:
+
+  ```
+sudo mkdir -p /etc/systemd/resolved.conf.d && sudo systemctl reboot
+  ```
+
+## [systemd crash with CoreOS stable 34.20210711.3.0](https://github.com/coreos/fedora-coreos-tracker/issues/912)
+
+  **Affected Versions:**  4.7.0-0.okd-2021-08-07-063045 - 4.7.0-0.okd-2021-08-07-063045
+
+  **Description:** Systemd version systemd-248.5-1.fc34.x86_64 may core dump with this message:
+
+  ```
+systemd[1]: Caught <ABRT>, dumped core as pid 3444.
+systemd[1]: Freezing execution.
+  ```
+  As an initial symptom, the node may have slowness and timeouts for interactive ssh and sudo. From the perspective of the Kubernetes scheduler, the nodes are available but pods created after ABRT report when using "get pod" ContainerCreating and when using "describe pod" a Status of Pending.
+
+  Because of [two CVEs](https://github.com/coreos/fedora-coreos-tracker/issues/904) package updates were fast-tracked. Afterwards, as noted in [an issue](https://github.com/coreos/fedora-coreos-tracker/issues/912) and [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1984651), systemd may respond as if sent an ABRT signal and crash. Differences between systemd-248.5 and 248.6 are listed at [Bodhi](https://bodhi.fedoraproject.org/updates/FEDORA-2021-3141f0eff1). A [commit](https://github.com/openshift/okd-machine-os/pull/181) to remove the package exceptions from okd-machine-os was made in mid-August 2021.
+
+  **Workaround:** Reset nodes as systemd crashes.
```

Comment:
Reset as in "reinstall"? iiuc a reboot is sufficient

Reply:
Reset as in, an external forced reboot of a VM, versus a graceful software reboot. Not as in re-install, no.

Reply:
Ah, hmm, lets change this to "reboot the machine" so that "node" would not be confused with k8s nodes

Reply:
All set (though I notice now I left it plural)